core/src/main/java/org/apache/brooklyn/enricher/stock/Transformer.java

```diff
@@ -44,9 +44,6 @@
     public static ConfigKey<Function<?, ?>> TRANSFORMATION_FROM_VALUE = ConfigKeys.newConfigKey(new TypeToken<Function<?, ?>>() {}, "enricher.transformation");
     public static ConfigKey<Function<?, ?>> TRANSFORMATION_FROM_EVENT = ConfigKeys.newConfigKey(new TypeToken<Function<?, ?>>() {}, "enricher.transformation.fromevent");
     
-    public Transformer() {
```

Comment:
i like to include the explicit no-args constructor if it's intended; no harm in removing it but no point either....